### ISSUE TITLE
Thread gets posted with a different account

### ISSUE THREAD

**Issue by heyitsellecross** [simonetthomas/Blue_Thread#39]
**Describe the bug**
A thread has been posted using a different account

**To Reproduce**
1. I was and currently still logged-in Blue Thread, using my own app password

2. Another user, User B, logs in Blue Thread, (using their own app password)

3. When they post using Blue Thread, the post shows up being posted using my account instead of User B. 

**Expected behavior**
The posts of User B should be posted with their own account, and not mine. 

**Screenshots**
NA

**Additional context**
Please refer to these posts in Bluesky: 

A. https://bsky.app/profile/ellecross.bsky.social/post/3kbvlf34uta26 (start of the post) 
B. https://bsky.app/profile/asmoth360.bsky.social/post/3kbwd2y3vns2j (comment from Asmoth)

**Comment by palila7** [comment]
This bug happened to my account as well. Yesterday I logged into Blue Thread using an app password on Safari 17.0 for iOS, made a little test thread that posted to my Bluesky account as expected (I thought maybe it would post to ElleCross’s account since I was trying to recreate the issue, but for whatever reason it didn’t happen that time), then I deleted my test thread. I stayed logged in to Blue Thread overnight though and this morning noticed a thread from another user got posted to my account.
First post of the thread that got posted to my account:
https://bsky.app/profile/palila.bsky.social/post/3kbw5xipznk2g
For now, I’ve deleted the app password that I used for Blue Thread. Thanks for taking a look at this bug Asmoth360!
